### PR TITLE
Register Neo4j [https://github.com/glesica/Neo4j.jl.git]

### PR DESCRIPTION
**Pull request by glesica**
This is a driver library for the Neo4j graph database.

### CODE REVIEW

File: Neo4j/versions/1.0.0/requires [hunk 1]
```diff
@@ -1,3 +1,3 @@
-julia 0.4-
+julia 0.4
```
Comment (tkelman) [inline Neo4j/versions/1.0.0/requires]: do this on master of the package too

**Comment by tkelman** [comment]
you're missing the dependency on JSON in REQUIRE

**Comment by glesica** [comment]
@tkelman OK, I think we're good here.

**Comment by tkelman** [comment]
Yep, LGTM. Will leave this open a total of 24 hours in case anyone else has any initial feedback.

**Pull request merged**